### ISSUE TITLE
Local fallback breaks italics

### ISSUE THREAD

**Issue by RussellSprouts** [alfredxing/brick#41]
I have a site that uses EB Garamond, and on my machine italics do not work. I have EB Garamond installed locally. The local fallback for EB Garamond loads the "EB Garamond 12 Regular" font, so nothing is italicized.

``` css
@font-face{
    font-family:'Raleway';
    font-style:normal;
    font-weight:300;
    src:local('Raleway Light'),url(//brick.a.ssl.fastly.net/fonts/raleway/300.woff) format('woff')
}

@font-face{
    font-family:'Raleway';
    font-style:italic;
    font-weight:300;
    src:local('Raleway Light'),url(//brick.a.ssl.fastly.net/fonts/raleway/300i.woff) format('woff')
}

@font-face{
    font-family:'EB Garamond';
    font-style:normal;
    font-weight:400;
    src:local('EB Garamond 12 Regular'),url(//brick.a.ssl.fastly.net/fonts/ebgaramond/400.woff) format('woff')
}

@font-face{
    font-family:'EB Garamond';
    font-style:italic;
    font-weight:400;
    src:local('EB Garamond 12 Regular'),url(//brick.a.ssl.fastly.net/fonts/ebgaramond/400i.woff) format('woff')
}

@font-face{
    font-family:'EB Garamond';
    font-style:normal;
    font-weight:700;
    src:local(''),url(//brick.a.ssl.fastly.net/fonts/ebgaramond/700.woff) format('woff')
}

@font-face{
    font-family:'EB Garamond';
    font-style:italic;
    font-weight:700;
    src:local(''),url(//brick.a.ssl.fastly.net/fonts/ebgaramond/700i.woff) format('woff')
}
```

**Comment by alfredxing** [comment]
Wow. Nice catch! I can't believe I let a bug of this size slip through. Just fixed in b63830d947e4794d7541ee67194afa901e2dee5d. Let me know if this works for you.

Also, a side note that EB Garamond doesn't come in bold.

**Comment by RussellSprouts** [comment]
Yep, it looks good for me. I've also removed the reference to EB Garamond 700 from my stylesheet import, so it won't try to load it. Thanks!
